.travis.yml

```diff
@@ -23,12 +23,9 @@ addons:
     # SAUCE_ACCESS_KEY=<secure>
     secure: hHF06xHY6pLdxscwYhUHdSRuUH2CPA61AUrvPiW54lnMwwkDvCQakR+aY5HbjptFIfOwkWsS4xoqXH4pZTfhMOji32S/+ELT+AqLdkIgnLYDfBj4RcsZoXyvuiLx29i4PVCnGMikctBp4dM6wORgfd1tu3uwhcoQrBjxCMW1qmY=
 cache:
+  yarn: true
```

Comment:
The cache dir was explicit because Travis caches the wrong directory for `yarn: true` (`~/.yarn-cache` or something), though that may be fixed